assets/src/edit-story/components/header/postPublishDialog.js

```diff
@@ -54,18 +52,30 @@ function PostPublishDialog({ open, onClose, confirmURL, storyURL }) {
       title={__('Story published!', 'web-stories')}
       actions={
         <>
-          <Plain onClick={onClose}>{__('Dismiss', 'web-stories')}</Plain>
-          <Plain href={confirmURL} onClick={onAddToPostClick}>
+          <Button
+            type={BUTTON_TYPES.TERTIARY}
+            size={BUTTON_SIZES.SMALL}
+            onClick={onClose}
+          >
+            {__('Dismiss', 'web-stories')}
+          </Button>
+          <Button
+            type={BUTTON_TYPES.PRIMARY}
+            size={BUTTON_SIZES.SMALL}
+            href={confirmURL}
+            onClick={onAddToPostClick}
+          >
```

Comment:
💭 Should this be using the new dialog in the editor that was added? `assets/src/edit-story/components/dialog/dialog.js`

Feels like we don't need to re-do this dialog here too. I like what you did in `assets/src/edit-story/components/header/previewErrorDialog.js`

Reply:
This happens in one other place too. Is there a reason why it's different in these files?

`assets/src/edit-story/components/header/postPublishDialog.js`
`assets/src/edit-story/components/header/titleMissingDialog.js`

Reply:
The second button has an `href` here, but I assume it also needs an `as="a"` to actually be a link?

Reply:
`Button` is checking for a `href` and changing itself to an anchor, so the `as="a"` isn't needed. 
![Screen Shot 2021-03-16 at 11 14 45 AM](https://user-images.githubusercontent.com/10720454/111359534-dab51580-8648-11eb-85dd-1217f2bca0e6.png)
![Screen Shot 2021-03-16 at 11 14 53 AM](https://user-images.githubusercontent.com/10720454/111359539-db4dac00-8648-11eb-9dab-6cb65e978089.png)

This `Dialog` is internal to the editor, so that the theme can get inverted. 

I left the buttons in tact because of the href. It was starting to feel redundant to take this into a shared dialog when it's nice to have access to what the action/structure should be within the context of the component.